docs/sources/mimir/release-notes/v2.9.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Grafana Mimir version 2.9 release notes"
+menuTitle: "V2.9 release notes"
+description: "Release notes for Grafana Mimir version 2.9"
+weight: 300
+---
+
+# Grafana Mimir version 2.9 release notes
+
+Grafana Labs is excited to announce version 2.9 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Experimental support for optimized series selection strategy** Reduce the volume of data necessary to complete requests with individual label selectors that are present on a lot of series.
+- **Reduced CPU utilisation for some shuffle sharding scenarios** Mimir queriers will now use significantly less CPU in cases where shuffle sharding is enabled with a high number of shards.
+- **Reduced object storage API calls in compactors and rulers** Mimir 2.9 comes with optimizations that will reduce the amount of times compactors and rulers need to access rules stored in object storage.
```

Comment:
We need to expand here. I would include:
- Added experimental ruler storage cache support. The cache should reduce the number of "list objects" API calls issued to the object storage when there are 2+ ruler replicas running in a Mimir cluster. The cache can be configured setting `-ruler-storage.cache.*` CLI flags or their respective YAML config options.
- Introduced a new feature to trigger a synchronization of tenant's rule groups as soon as they change the rules configuration via API. This synchronization is in addition of the periodic syncing done every `-ruler.poll-interval`. The new behavior is enabled by default, but can be disabled with `-ruler.sync-rules-on-changes-enabled=false` (configurable on a per-tenant basis too).

Reply:
I added two bullet points for the individual enhancements.